nfc/CHANGELOG.rst

```diff
@@ -30,6 +30,8 @@ Bug fixes
 
  * Fixed handling R(NAK) frame after sending R(ACK) frame.
    The Tag should respond with last transmitted R(ACK) frame.
+ * Fixed the race condition that occurred when the WTX frame is scheduled and the library tries to send a data chunk.
```

Comment:
```suggestion
 * Fixed the race condition that occurred when the WTX frame was scheduled and the library tried to send a data chunk.
```

Reply:
Sorry, I should have noticed this earlier.

Reply:
Mia will fix this in https://github.com/nrfconnect/sdk-nrfxlib/pull/642.

Reply:
Fixed now.